app/controllers/predicates/AuthPredicate.scala

```diff
@@ -65,20 +65,20 @@ class AuthPredicate @Inject()(authService: EnrolmentsAuthService,
           if (enrolments.enrolments.exists(_.key == AuthKeys.agentEnrolmentId)){
             authoriseAsAgent(block)
           } else {
-            logger.debug("[AuthPredicate][invokeBlock] - Agent does not have correct agent enrolment ID")
+            debug("[AuthPredicate][invokeBlock] - Agent does not have correct agent enrolment ID")
             Future.successful(Forbidden(unauthorisedAgent()))
           }
 
         case Some(_) ~ enrolments => authoriseAsNonAgent(enrolments, block)
         case None ~ _ =>
-          logger.warn("[AuthPredicate][invokeBlock] - Missing affinity group")
+          errorLog("[AuthPredicate][invokeBlock] - Missing affinity group")
           Future.successful(errorHandler.showInternalServerError)
       } recover {
         case _: NoActiveSession =>
-          logger.debug(s"[AuthPredicate][invokeBlock] - No active session. Redirecting to ${appConfig.signInUrl}")
+          errorLog(s"[AuthPredicate][invokeBlock] - No active session. Redirecting to ${appConfig.signInUrl}")
           Redirect(appConfig.signInUrl)
         case _: AuthorisationException =>
-          logger.warn("[AuthPredicate][invokeBlock] - Unauthorised exception when retrieving affinity and all enrolments")
```

Comment:
same here

Reply:
i think this should be kept as errorLog since the user is not authorised to proceed with the journey